ui/src/app/pages/login/account-creation/account-creation-tos.spec.tsx

```diff
@@ -71,14 +48,15 @@ it('should enable checkboxes and next button with user input', async() => {
   expect(getNextButton(wrapper).prop('disabled')).toBeTruthy();
 
   // Now, simulate checking both boxes, which should enable the "next" button.
+  debugger;
   getPrivacyCheckbox(wrapper).simulate('change', {target: {checked: true}});
```

Comment:
Per slack discussion, it appears that in order to trigger the event property you need to call `simulate()` on the `input` rather than the `CheckBox` element.